src/app/shared/search/search-filters/search-filters.component.ts

```diff
@@ -59,8 +60,9 @@ export class SearchFiltersComponent implements OnInit, OnDestroy {
    * Link to the search page
    */
   searchLink: string;
-
+  searchConfig: SearchConfig;
   subs = [];
+  filterLable = 'search';
```

Comment:
Small misspelling in this code.  "filterLable" should be spelled "filterLabel".  (This same variable is used elsewhere in the code, so you'll want to copy this change to other classes that use it)

Reply:
Again, this has a small misspelling.  `filterLable` should be `filterLabel`